### PR TITLE
tmedia-614-tap-targets

### PR DESCRIPTION
**Pull request by brentmiller1973**
## Description
Promo's that are children of the top table block are not providing enough of a tap target on mobile

## Jira Ticket
[- [TMEDIA-](https://arcpublishing.atlassian.net/browse/TMEDIA-)](https://arcpublishing.atlassian.net/browse/TMEDIA-614)

## Acceptance Criteria
Acceptance criteria
On all-blocks page, there should be no tap target issues between the overlapping headline and overline
This should only affect the mobile version

## Test Steps
1. Checkout this branch `git checkout tmedia-614-tap-targets`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/shared-styles,@wpmedia/extra-large-manual-promo-block,@wpmedia/extra-large-promo-block,@wpmedia/large-manual-promo-block,@wpmedia/large-promo-block,@wpmedia/medium-manual-promo-block,@wpmedia/medium-promo-block,@wpmedia/small-manual-promo-block,@wpmedia/small-promo-block,@wpmedia/video-promo-block,@wpmedia/top-table-list-block
`
3. Bring up published version of an all blocks page.
4. Run SEO on lighthouse.
5. Verify no tap target issues are occuring on any promo block either stand alone or inside a top table.
6. Ignore tap target issues for links that are in random inserted / embedded HTML.

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [x ] Confirmed all the test steps a reviewer will follow above are working.
- [x ] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x ] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [ x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x ] Confirmed this PR has unit test files
  - [x ] Ran `npm run test`, made sure all tests are passing

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/shared-styles/scss/_large-promo.scss [hunk 1]
```diff
@@ -9,9 +9,11 @@
     display: inline-block;
     font-size: calculateRem(16px);
     line-height: calculateRem(24px);
-    padding-bottom: map-get($spacers, 'sm');
+    padding-top: map-get($spacers, 'xs');
```
Comment (JackHowa) [inline blocks/shared-styles/scss/_large-promo.scss:12]: looks like this is affecting the desktop breakpoint? in the acceptance criteria, this should only affect the mobile viewport

**Review by JackHowa — CHANGES_REQUESTED** [review]
It appears that this is affecting the desktop viewport. In the acceptance criteria, the goal is to only change styles associated with the mobile viewport

File: blocks/shared-styles/scss/_large-promo.scss (hunk 1 above)
Comment (brentmiller1973) [reply blocks/shared-styles/scss/_large-promo.scss:12]: @JackHowa Consolidated the change to affect mobile only.

**Review by JackHowa — DISMISSED** [review]
Thanks for adding padding-bottom back to the desktop styles for overline large promo within the media query. 

Tap targets and web vitals are tough. And I hope the criteria makes sense as written. But I'm seeing the web vitals tap targets failing for the http://localhost/pf/all-blocks/?_website=dagen and http://localhost/pf/all-blocks/?_website=the-sun. I wonder if we're looking at different pages or sites. In the pr, there was mention of an all-blocks page. If the ac needs to be clarified or design needs to be consulted, let me know and we can go through it. 

Screen Shot 2022-02-11 at 10 54 06

Screen Shot 2022-02-11 at 09 58 44

**Comment by brentmiller1973** [comment]
@JackHowa Good catch, I was testing with the large promos in top table list block.  I needed to account for when the promo is not in one.  Change have been pushed.

**Comment by nschubach** [comment]
I am getting this: 
image

http://localhost/pf/all-blocks-jc/?_website=arc-demo-2
Mobile lighthouse test

Seems like the `xs` is not enough here?
image

**Comment by brentmiller1973** [comment]
> Seems like the `xs` is not enough here? 

@nschubach Thanks for this, wasn't the version of the all blocks page I was testing on so I missed this.  I have updated the padding for both the large and extra large promos and they are now passing on http://localhost/pf/all-blocks-jc/?_website=arc-demo-2. Also, you can ignore the below tap target issues, they do not apply to promos:

image

**Review by nschubach — APPROVED** [review]
Looks like it's meeting the requested changes.  Can't find any other instances.

**Pull request merged**